### PR TITLE
#4213 translation string fixes

### PR DESCRIPTION
**Pull request by GalacticChimp**
## About the PR 
Fixes https://github.com/space-wizards/space-station-14/issues/4213
Requires https://github.com/space-wizards/RobustToolbox/pull/1841

### CODE REVIEW

**Comment by GalacticChimp** [comment]
Calling @SethLafuente @mirrorcult @metalgearsloth since they reported some of the issues

File: Resources/Prototypes/Entities/Objects/Power/antimatter_part.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   parent: BaseItem
   id: AMEPart
   name: AME part
-  description: A flatpack used for constructing an antimatter engine reactor.\nUse a multitool to unpack it.
+  description: A flatpack used for constructing an antimatter engine reactor. Use a multitool to unpack it.
```
Comment (GalacticChimp) [inline Resources/Prototypes/Entities/Objects/Power/antimatter_part.yml:5]: \n doesn't seem to work when placed in description field. I didn't change anything related to item descriptions in my original PR, so I'm not sure why it shows as a literal \n on examine

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Objects/Power/antimatter_part.yml:5]: It's probably a mistaken addin from byond.

**Comment by GalacticChimp** [comment]
this one is ready for merge I think

**Pull request merged**